docs/tasks.md

```diff
@@ -241,7 +241,24 @@ steps:
     #!/usr/bin/env bash
     /bin/my-binary
 ```
+#### Specifying a timeout
 
+A step can specify a `timeout` field. If the `Step` execution time exceeds the specified timeout, the `Step` and consequently the entire `TaskRun` is canceled.
+An accompanying error message is output under `status.conditions.message`.
+
+The format for a timeout is a duration string as specified in the [Go time package](https://golang.org/src/time/format.go?s=40541:40587#L1364) (e.g. 1s or 1ms).
```

Comment:
Could we link to https://golang.org/pkg/time/#ParseDuration instead? That has roughly the same information, and is less likely to fall out of sync than a link to specific lines of code in the Go stdlib.